00-programming-fundamentals/exercises/merge-conflict-remote.md

```diff
@@ -0,0 +1,102 @@
+
+In this activity you'll be working with your fellow Adies to put together multiple scraps from a single recipe using Git. You'll accomplish this by successively merging in content from each individuals' copy of the recipes repo. Once all the of merges are finished (and any merge conflicts resolved) you'll have a complete recipe document which can be submitted to the original repository with a pull request.
+
+## Learning Goals
+
+The goal of this activity is to give you exposure to the process of resolving merge conflicts. Because merge conflicts are a natural part of any collaborative development effort, this is an important skill to have as a developer.
+
+With this activity you will be able to encounter and resolve merge conflicts in a controlled scenario where the correct resolution of all merge conflicts is clear.
+
+## Instructions
+
+### Baseline Setup
+
+Before we can start putting the recipe together we need to make sure that everyone in your group is working from the same starting point and can effectively contribute to the recipe building effort.
+
+1. **Split into groups**: The class will be divided into groups using the breakout sections feature of Zoom, these assignments are random.
+   1. Breakout Room 1:  Aardvark
+   2. Breakout Room 2:  Tarsier
+   3. Breakout Room 2:  Gecko
+   4. Breakout Room 2:  Sloth
+   5. Breakout Room 2:  Lionfish
+   6. Breakout Room 2:  Pangolin
+   7. Breakout Room 2:  Rhino
+   8. Breakout Room 2:  Curassow
+2. **Fork the activity repo**: One person in your group should fork the carnival-recipes repo and then add everyone else as a collaborator.
+3. **Clone the forked repo**: Everyone else in your group should clone the forked repository.
+4. **Find your recipe scrap**: Each group is assigned six recipe scraps that should all be part of the same recipe. You can split these scraps up amongst your group however you wish. The scraps assigned to each group are listed in the carnival-recipes repo. If your group has fewer than six people someone may need to take multiple scraps.
+5. **Add your recipe scrap**: Copy the contents of your recipe scrap into the recipe file and then add and commit your changes to the recipe file. After completing this step each person in your group should have **different** content within the recipe file in their local repository.
+
+**Do not** push your changes to GitHub yet! If you're finished early with this step help your groupmates.
+
+### Building the recipe
+
+Once everyone in your group has completed the baseline setup instructions above you're ready to start building the recipe together.
+
+#### General advice
+
+There are a few things to look out for when working through this activity:
+
+- If you're still unsure about what a merge conflict is, or how to resolve one please review the [Merge Conflicts curriculum](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/git-merge-conflicts.md).
```

Comment:
I'm trying to remember if this is a reading we've already assigned them. If not, we should probably assign it to them the day before this activity.
Even if we have, it might be worth putting a thing in the hw the day before about reviewing this reading.

Reply:
It's in the calendar

Reply:
Oh, ok.